### ISSUE TITLE
ASPNET CORE MVC not working with External DotNet Standard Library

### ISSUE THREAD

**Issue by Phoenigore** [dotnet/core#1184]
# Issue Title

ASPNET CORE MVC not working with External DotNet Standard Library

# General

When you want to use a DotNet Standard Library 2.0 project into AspNet Core MVC Project from another solution, the AspNetCore MVC program crash with this error.

InvalidOperationException: Cannot find compilation library location for package 'Name.Of.DotNetStandard2.0Project'
Microsoft.Extensions.DependencyModel.CompilationLibrary.ResolveReferencePaths(ICompilationAssemblyResolver resolver, List<string> assemblies).

The same test with an AspNet MVC 4.6 Framework project is a success

Regards

**Comment by dasMulli** [comment]
How do you reference the library? a project reference or a dll reference?

**Comment by nefcanto** [comment]
I also get this annoying error.

> Cannot find compilation library location for package 'Holism.Framework'

I'm trying to reuse my `.cshtml` files in other projects.
I have a simple class library that includes some `.cshtml` files. I've added them as embedded resources according to this [SO question](https://stackoverflow.com/questions/47726991/how-to-reuse-views-pages-in-asp-net-core-mvc).

Then I've added an application part for that assembly to my MVC service configuration.

I've also added that class library's output DLL file as a dll reference to the MVC project. Yet when I want to access that view, I get the aforementioned error.
